front/src/modules/trainschedule/components/ManageTrainSchedule/Itinerary/DisplayVias.tsx

```diff
@@ -99,6 +100,12 @@ export default function DisplayVias({ zoomToFeaturePoint }: DisplayViasProps) {
                             `KM ${place.position && Math.round(place.position) / 1000}`
                           }`}
                         </small>
+                        <small className="">
+                          {place.ch === '00' || !place.ch ? 'BV' : `${place.ch}`}
```

Comment:
I see this pattern in multiple different places in this PR:
- testing if `00`
- then if empty
- then fallback to `BV`

Should we factorize this functionality in only one place (a function or something, I'm not sure what is the best abstraction in the front-end for that).

Reply:
same as above

Reply:
done

Reply:
Yes yes yes, but it should display nothing when BV/00